hedera-evm/src/main/java/com/hedera/services/ledger/accounts/AliasResolver.java

```diff
@@ -0,0 +1,31 @@
+package com.hedera.services.ledger.accounts;
```

Comment:
Should this interface be in a separate package?  That would ease module extraction once the interfaces are created.